### PR TITLE
Improves QA tag processing to better support local development

### PR DESCRIPTION
**Pull request by Stringy**
## Description

Fixes issues where the QA tag was not set correctly for local testing.

## Checklist
- [x] Investigated and inspected CI test results

If any of these don't apply, please comment below.

## Testing Performed

Tested locally, and verified that images are pulled correctly.

### CODE REVIEW

File: integration-tests/suites/config/images.go
Comment (Molter73) [inline integration-tests/suites/config/images.go:65]: [ultra-nit] Do we need lazy loading of this variable? If we only read it once or twice, can't we just read it always and make the function slightly simpler? 

Or maybe since it's used as part of the `ImageStore` struct we could create a `func (i *ImageStore) New()` that will initialize this value internally? I could even argue the existing `loadImageStore` function could be easily refactored into `New`.

**Review by JoukoVirtanen — APPROVED** [review]
I tested this and it works. You can merge once you get approval from Mauro.

File: integration-tests/suites/config/images.go
Comment (Stringy) [reply integration-tests/suites/config/images.go:65]: Yeah it's read a lot over the course of a test run so it's worth caching the result, but your point about `loadImageStore` is good so I'll move it there

**Review by Molter73 — APPROVED** [review]
LGTM!

**Pull request merged**
